components/moderation-panel/ModEditFacilityOrProfessionalTopbar.vue

```diff
@@ -87,20 +87,19 @@
                         </button>
                     </div>
                     <div v-if="modalType === ModalType.DeleteConfirmation">
-                        <div
-                            v-show="moderationScreenStore.editFacilityScreenIsActive()"
-                            class="font-bold text-3xl"
-                        >
-                            {{ t('modEditFacilityOrHPTopbar.deleteConfirmationFacility',
-                                 { id: selectedId, facility: facilitiesStore.selectedFacilityData?.nameEn }) }}
+                        <div class="font-bold text-3xl">
```

Comment:
instead of having some much display logic in the javascript, we could just make two separate divs and conditionally `vi-if` show them. I think this would make this a bit more simple